userspace/libsinsp/utils.h

```diff
@@ -434,3 +436,26 @@ inline void hash_combine(std::size_t &seed, const T& val)
 // Log helpers
 ///////////////////////////////////////////////////////////////////////////////
 void sinsp_scap_debug_log_fn(const char* msg);
+
+///////////////////////////////////////////////////////////////////////////////
+// Set operation functions.
+///////////////////////////////////////////////////////////////////////////////
+
+
+template<typename T>
+std::set<T> unordered_set_to_ordered(std::unordered_set<T> unordered_set);
+
+template<typename T>
+std::unordered_set<T> convert_vector_to_unordered_set(std::vector<T> v);
```

Comment:
```suggestion
std::unordered_set<T> vector_to_unordered_set(const std::vector<T>& v);
```
I'm suggesting to remove the `convert_` prefix just to be coherent with the name of the other helpers. Not strong on this aspect, the important part is the const reference!